clusters/build-clusters/01_cluster/README.md

```diff
@@ -120,6 +120,35 @@ It is automated by config-updater:
 * alertmanager secret for sending notification to slack
 * PVCs for monitoring stack
 
+## Upgrade the cluster
+
+### Updating between minor versions
+
+Upgrade channel configuration, e.g., from OCP 4.3 to 4.4
+
+```
+oc --as system:admin --context build01 patch clusterversion version -p '{"spec":{"channel":"candidate-4.4"}}'
```

Comment:
`candidate-*` are not supported.  Not clear to me where build01 sits in the "bleeding edge" vs. "stable/supported" continuum, but maybe link [here][1] or some such for folks who want to understand the channel semantics.

[1]: https://docs.openshift.com/container-platform/4.3/updating/updating-cluster-between-minor.html#understanding-upgrade-channels_updating-cluster-between-minor

Reply:
OK ... Updated with `stable-4.4`.

@stevekuznetsov changed to `candidate-4.3` because we wanted to try the nightly builds with fixes.

If this is still true in the future, we can change it to `candidate-4.4` then.

Reply:
> ... to try the nightly builds with fixes.

Nightlies aren't in any channels.  From [the docs][1], the hierarchy is:

* Nightlies.  No updates in or out supported or recommended.  But you do get bleeding-edge features without waiting on ART.
* Candidates: Not supported, but you do get update recommendations via Cincinnati.
* Fast: Supported. No update-recommendation delay, so support might sometimes be... exciting.  But RH commits to getting you out.
* Stable: Supported, and update-recommendations are delayed, so should be nice and boring (although you have to wait longest for a given fix to land).

[1]: https://docs.openshift.com/container-platform/4.3/updating/updating-cluster-between-minor.html#understanding-upgrade-channels_updating-cluster-between-minor

Reply:
OK ... So for us, we probably should stay with `stable` channels and try nightly with the image SHA as we do it in the script.